packages/demand-bloc-track-trip/src/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { SearchParams } from '@karhoo/demand-api'
+import { TripStatuses } from './statuses'
+
+export const makeSearchParams = (
+  statuses: TripStatuses[],
+  paginationOptions?: Pick<SearchParams, 'pagination_offset' | 'pagination_row_count'>
+): Partial<SearchParams> => {
+  return {
+    trip_states: statuses,
+    pagination_offset: paginationOptions?.pagination_offset || 0,
+    pagination_row_count: paginationOptions?.pagination_row_count || 2,
+    order_by: ['date'],
+  }
```

Comment:
`order_by` could contain other options of sorting (see [swagger](https://github.com/karhoo/api-v1-bookings/blob/818adaf772927457cbecf6ba8ccee839cc367dc5/docs/swagger/bookings.yaml#L1752)). I'd propose to define enum for it